tuplex/python/src/PythonBindings.cc

```diff
@@ -35,7 +35,7 @@ PYMODULE {
     // std::cout<<"init done"<<std::endl;
 
 
-    register_exception_translator<tuplex::PythonException>(&tuplex::translateCCException);
+    //register_exception_translator<tuplex::PythonException>(&tuplex::translateCCException);
```

Comment:
Remove or leave a comment as to why this is commented out (so we don't forget in the future).

Reply:
done.